spec/liquid_tags/github_tag/github_readme_tag_spec.rb

```diff
@@ -7,6 +7,7 @@
     let(:url_repository_not_found) { "https://github.com/abra/cadabra" }
     let(:path_repository) { "rust-lang/rust" }
     let(:repo_owner) { "rust-lang" }
+    let(:relative_link) { "<a href=\"https://github.com/rust-lang/rust#installing-from-source\">Installing From Source</a>" }
```

Comment:
Hum... I'm not sure why this one doesn't match:

```html
<p>The build triple can be specified by either specifying <code>--build=&lt;triple&gt;</code> when
invoking <code>x.py</code> commands, or by copying the <code>config.toml</code> file (as described
in <a href="#installing-from-source">Installing From Source</a>), and modifying the
<code>build</code> option under the <code>[build]</code> section.</p>
```

Reply:
That's because it's using a pre-recorded cassette, you can change the name of the cassette in `VCR.use_cassette()`, this way it records a new one for the repository with the relative URL

Reply:
OK, I think I found why.

The resulting URL was `https://github.com/rust-lang/rust/#installing-from-source` instead of `https://github.com/rust-lang/rust#installing-from-source` (trailing slash).

Reply:
I'm wondering if we only fetch/display the first few kb of the readme (and end with "view on Github") 

https://github.com/forem/forem/blob/main/app/views/liquids/_github_readme.html.erb#L15-L20 it looks like we do this when we render the partial (the HTML_Truncator.truncate call in the view). By the time we get the parsed liquid tag back the preview is already a shortened view of the full readme (so the link toward the bottom of the page which _would_ have rendered the link you were looking for wasn't shown yet).

Reply:
Oh okay, so we cannot use rust-lang/rust...

Reply:
Should I go back to using another repostiory?

Reply:
My vote goes to:

1. repository with relative URLs (the intial one)
2. new cassette

Reply:
Ok! I'll take care of it in a few hours.